packages/salesforcedx-sobjects-faux-generator/src/describe/configUtil.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  Aliases,
+  ConfigAggregator,
+  ConfigFile,
+  ConfigValue
+} from '@salesforce/core';
+import * as path from 'path';
+
+const defaultUserNameKey = 'defaultusername';
+
+export class ConfigUtil {
+  public static async getUsername(
+    projectPath: string
+  ): Promise<string | undefined> {
+    const defaultUserName = (await this.getConfigValue(
+      projectPath,
+      defaultUserNameKey
+    )) as string;
+    const username = await Aliases.fetch(defaultUserName);
+    return Promise.resolve(username);
+  }
+
+  public static async getConfigValue(
```

Comment:
Is there an easy way to reuse [this](https://github.com/forcedotcom/salesforcedx-vscode/blob/3ff4f374e491c4d95f6c4b33b8c4fb5dc3930990/packages/salesforcedx-vscode-core/src/util/configUtil.ts#L38)?

Reply:
It's not easy, I initially looked at having this code in the `salesforcedx-utils-vscode` module but that caused compilation issues in `salesforcedx-vscode-core` and `salesforcedx-vscode-apex`. The origin of the problems seems to be with importing more than one module that depends on `@salesforce/core`. Fixing/refactoring this would be a much bigger work item that was out of scope for this fix.